presto-main/src/main/java/io/prestosql/SystemSessionProperties.java

```diff
@@ -116,6 +116,7 @@
     public static final String IGNORE_STATS_CALCULATOR_FAILURES = "ignore_stats_calculator_failures";
```

Comment:
I am afraid that information about the original author of this commit got lost.

Reply:
Yes, I couldn't preserve original author info in this commit because the patch from prestodb could not be applied cleanly to prestosql code due to changes in package names and import ordering. In case there is something I can do to preserve that info, please let me know.

Reply:
@raunaqmorarka when you "lose" commit authorship during rebase, you can restore it with the following
```
git commit --amend --author='Andrii Rosa <andriirosa@fb.com>' -C HEAD
```
(This assumes the problematic commit is the topmost one, as it is in this case. Otherwise you would `git rebase -i` to edit the affected commit)

Reply:
Done